core/src/main/scala/chisel3/Aggregate.scala

```diff
@@ -1246,15 +1268,10 @@ package experimental {
   class BundleLiteralException(message: String) extends ChiselException(message)
   class VecLiteralException(message: String) extends ChiselException(message)
 
-  /** Indicates that the compiler plugin should generate [[cloneType]] for this type
-    *
-    * All user-defined [[Record]]s should mix this trait in as it will be required for upgrading to Chisel 3.6.
+  /** Indicates that the compiler plugin should generate [[clonType]] for this typll user-defined [[Record]]s should mix this trait in as it will be required for upgrading to Chisel 3.6.
     */
```

Comment:
```suggestion
```
Since it's deprecated, I think we can just delete the doc (since the doc is also wrong).